simulert/cli.py

```diff
@@ -0,0 +1,73 @@
+import os
```

Comment:
```suggestion
from pathlib import Path
```
This is for a suggestion further down.